Packs/CyberArkPAS/Integrations/CyberArkPAS/CyberArkPAS.py

```diff
@@ -361,7 +379,7 @@ def add_account(self,
                     safe_name: str,
                     password: str,
                     secret_type: str,
-                    properties: str,
+                    properties: dict,
```

Comment:
```suggestion
                    properties: dict | str,
```
right?

Reply:
It must be a dict, and the `order_properties_to_dict` function forces it to be a dict
(But following the comment I changed the typing for the `add_account_command` function)